docs/en/notes/changelog.md

```diff
@@ -1,5 +1,55 @@
 # Changelog of v1.1
 
+### v1.1.1 (26/5/2023)
+
+#### Highlights
+
+- Support [TPVFormer](https://arxiv.org/pdf/2302.07817.pdf) in `projects` (#2399, #2517, #2535)
+- Support the training of \[BEVFusion\] in `projects` (#2546)
+- Support Lidar-based 3D Semantic Segmentation Benchmark (#2530, #2559)
+
+#### New Features
+
+- Support [TPVFormer](https://arxiv.org/pdf/2302.07817.pdf) in `projects` (#2399, #2517, #2535)
+- Support the training of \[BEVFusion\] in `projects` ()
+- Support LiDAR-based 3D Semantic Segmentation Benchmark (#2530, #2559)
+- Support test-time augmentation for Segmentor (#2382)
+- Support Minkowski ConvModule and Residual Block (#2528)
```

Comment:
Please add a line:
- Support the visualization of multi-view images in multi-modal methods.